backend/sqlmapper/drivers/pg.go

```diff
@@ -171,3 +172,30 @@ func (s *pgStore) FindByColumnName(columnName string, value string, offset int,
 
 	return sqlmapper.RowsToQueryResults(rows, s.Columns)
 }
+
+func (s *pgStore) Update(d sqlmapper.RowData, id int) (sqlmapper.RowData, error) {
+	// TODO: verify column in data-set is correct, check rowData is empty, check primary key is not exist
+
+	if err := verifyInput(&d, s.TableName, s.ModelList); err != nil {
```

Comment:
In case update should check it have ID also raise a error when case create have ID